cmake/FindGeant4.cmake

```
@@ -1,13 +1,18 @@
 # Find Geant4 cmake config file
 set(Geant4_SEARCH_DIRS)
-# Is this line necessary?
-set(PACKAGE_FIND_VERSION ${Geant4_FIND_VERSION})
+
 file(GLOB Geant4_SEARCH_DIRS "${GEANT4_DIR}/lib*/Geant4-*")
```

The new install paradigm does not include this hyphen
```suggestion
file(GLOB Geant4_SEARCH_DIRS "${GEANT4_DIR}/lib*/Geant4*")
```